controller/src/main/java/org/jboss/as/controller/SimpleResourceDefinition.java

```diff
@@ -441,16 +441,7 @@ public boolean isRuntime() {
      */
     @SuppressWarnings("deprecation")
```

Comment:
Nit, only if you touch this for some reason: The SuppressWarnings looks to be cruft.